examples/assets_machine_learning/machine_learning_example/README.md

```diff
@@ -0,0 +1,23 @@
+# machine_learning_example
+
+This is an example of how to use Dagster for machine learning. A full guide on [how to build machine learning pipelines with Dagster](https://docs.dagster.io/guides/dagster/ml-pipeline) can be used along with this example. 
+
+## Getting started
+
+```bash
+pip install -e ".[dev]"
+```
+
+Then, start the Dagster UI web server:
+
+```bash
+dagster dev
+```
+
+In this example, we use the Hacker News titles to predict the number of comments on a story. 
```

Comment:
Not everyone will know what Hacker News is, so maybe worth mentioning that it's an aggregator of technical news and linking to it.